RedHatInsights/rhsm-subscriptions
# SWATCH-587: use tally_instance_view to get records for instance api

https://issues.redhat.com/browse/SWATCH-587

Test Steps:

- Create test hosts: `bin/insert-mock-hosts   --num-guests=1   --num-hypervisors=1   --hypervisor-id=eb0d994a-e129-4a18-860d-3e39294f95df`
- Start app with: `DEV_MODE=true ./gradlew :bootRun`
- Run curl and should get records with VIRTUAL, PHYSICAL and HYPERVISOR measurement categories:
`curl -X 'GET'   'http://localhost:8000/api/rhsm-subscriptions/v1/instances/products/RHEL'   -H 'accept: application/json'   -H 'x-rh-identity: eyJpZGVudGl0eSI6eyJhY2NvdW50X251bWJlciI6ImFjY291bnQxMjMiLCJ0eXBlIjoiVXNlciIsInVzZXIiOnsiaXNfb3JnX2FkbWluIjp0cnVlfSwiaW50ZXJuYWwiOnsib3JnX2lkIjoib3JnMTIzIn19fQo='`
